ckan/requirements.in

```diff
@@ -8,21 +8,26 @@ git+https://github.com/GSA/ckanext-saml2auth.git@create_user_via_saml#egg=ckanex
 -e git+https://github.com/ckan/ckanext-archiver.git@master#egg=ckanext-archiver
 -e git+https://github.com/ckan/ckanext-report.git@master#egg=ckanext-report
 
-ckanext-datagovcatalog>=0.0.3
-ckanext-datagovtheme>=0.1.22
-ckanext-datajson>=0.1.13
-ckanext-envvars>=0.0.2
-ckanext-geodatagov>=0.1.28
-ckanext-googleanalyticsbasic
-ckanext-metrics-dashboard
+# ckanext-datagovcatalog>=0.0.3
+# ckanext-datagovtheme>=0.1.22
+# ckanext-datajson>=0.1.13
+ckanext-envvars>=0.0.3
+# ckanext-geodatagov>=0.1.28
+# ckanext-googleanalyticsbasic
+# ckanext-metrics-dashboard
+-e git+https://github.com/gsa/ckanext-geodatagov.git@ckan-2.10#egg=ckanext-geodatagov
```

Comment:
these should revert to new 2.10 published versions

Reply:
Yes it should. We will correct it once `ckan-210` is merged into the `main`. again i am surprized the existing changes in `ckan-210` shows up here in a `git rebase main` result.